### ISSUE TITLE
This should totally be PR'ed to the main stedolan/jq repository to replace the Dockerfile there

### ISSUE THREAD

**Issue by espoelstra** [kamermans/docker-jq#1]
The last build he had was ages ago and doesn't have this amazing functionality. It would be awesome to have an option to use a `master` branch build as well as the specific released versions. Pushes to docker hub could key off of commits to master and add the master build as the default `jq` binary with the option to override via the environment variable or simply override the entrypoint with  `--entrypoint /bin/jq-1.x` instead of needing to pass an environment variable to select the version. This would also make it much easier for non-technical folks to test out fixes to bugs without forcing them to setup a compilation environment.

https://github.com/stedolan/jq/blob/master/Dockerfile
https://hub.docker.com/r/stedolan/jq/

**Comment by kamermans** [comment]
Yeah, makes sense to me :)  I would say, though, that I have been using the distro-provided `jq` lately since it's new enough to get the job done (in my case).  It would still make sense for him to have some lightweight, portable `jq` docker images like mine though.
